src/HIPAA-Security/rules/rds/hipaaSecurityRDSLoggingEnabled.ts

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { CfnDBInstance } from '@aws-cdk/aws-rds';
+import { IConstruct, Stack } from '@aws-cdk/core';
+
+/**
+ * RDS DB instances are configured to export all possible log types to CloudWatch - (Control IDs: 164.308(a)(3)(ii)(A), 164.308(a)(5)(ii)(C))
+ * @param node the CfnResource to check
+ */
+export default function (node: IConstruct): boolean {
+  if (node instanceof CfnDBInstance) {
+    const dbType = JSON.stringify(Stack.of(node).resolve(node.engine));
+    const dbLogs = JSON.stringify(
+      Stack.of(node).resolve(node.enableCloudwatchLogsExports)
+    );
+    if (dbLogs == undefined) {
+      return false;
+    }
+
+    if (dbType.includes('mariadb') || dbType.includes('mysql')) {
+      if (
+        !(
+          dbLogs.includes('audit') &&
+          dbLogs.includes('error') &&
+          dbLogs.includes('general') &&
```

Comment:
I could see users wanting to force this rule for `audit`, but be okay with leaving out `general` to save on cost.

Could be re-visited later on